### PR TITLE
CATROID-552 "color at x y( x , y )" sensor [UI only]

### PR DESCRIPTION
**Pull request by petbuer**
Introduce a new sensor function color at x y( x , y ). This ticket includes UI changes only.

Jira: https://jira.catrob.at/browse/CATROID-552

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [X] Include the name of the Jira ticket in the PR’s title
- [X] Include a summary of the changes plus the relevant context
- [X] Choose the proper base branch (*develop*)
- [X] Confirm that the changes follow the project’s coding guidelines
- [X] Verify that the changes generate no compiler or linter warnings
- [X] Perform a self-review of the changes
- [X] Verify to commit no other files than the intentionally changed ones
- [X] Include reasonable and readable tests verifying the added or changed behavior
- [X] Confirm that new and existing unit tests pass locally
- [X] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [X] Stick to the project’s gitflow workflow
- [X] Verify that your changes do not have any conflicts with the base branch
- [X] After the PR, verify that all CI checks have passed
- [X] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by gPathpp — CHANGES_REQUESTED** [review]
As discussed today, here are my suggestions (solution for the flaws within the function) for this PR: https://github.com/petbuer/Catroid/pull/1

Please consider merging these suggestions.
(Afterward, tailor your PR to the requirements of the ticket. (Strings, etc..)) 

Nonetheless, great work!

**Review by gPathpp — APPROVED** [review]
Well done!
I think the language-test is not necessary, but please discuss this with @wslany.

File: catroid/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1833,6 +1833,8 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_function_collides_with_color_parameter" translatable="false">(\'#ff0000\')</string>
     <string name="formula_editor_function_color_touches_color">color touches color</string>
     <string name="formula_editor_function_color_touches_color_parameter" translatable="false">(\'#ff0000\',\'#00ff00\')</string>
+    <string name="formula_editor_sensor_color_at_x_y">color at x y</string>
+    <string name="formula_editor_sensor_color_at_x_y_parameter">(100,200)</string>
```
Comment (wslany) [inline catroid/src/main/res/values/strings.xml:1837]:
```suggestion
    <string name="formula_editor_sensor_color_at_x_y_parameter"  translatable="false">(100,200)</string>
```

**Review by wslany — COMMENTED** [review]
Thanks! @petbuer how far are you with CATROID-553? Because I think we should rather only merge this together with it, as we would otherwise need a new build flag just for this sensor to limit its visibility in release apks, which seems a bit excessive much to me. Also I saw that you assigned yourself already to it. PLMK.

**Comment by petbuer** [comment]
OK, I will notify you when CATROID-553 is ready. I just began to work on it.

**Comment by wslany** [comment]
> OK, I will notify you when CATROID-553 is ready. I just began to work on it.

Thanks. You can add it as commits to this same PR, i.e., make one PR for both tickets.

**Comment by wslany** [comment]
Thanks!

**Pull request merged**